src/CattyTests/Formula/FormulaElementTest.swift

```diff
@@ -30,8 +30,8 @@ final class FormulaElementTest: XCTestCase {
 
     override func setUp() {
         super.setUp()
-        let screenSize = Util.screenSize(true)
-        formulaManager = FormulaManager(sceneSize: screenSize, landscapeMode: false)
+        let stageSize = Util.screenSize(true)
```

Comment:
I'm not sure if "screenSize" should be renamed to "stageSize" - but please make sure to do it the same way in FormulaElementTest.swift and FormulaFunctionsTest.swift (below)